game_event/models.py

```diff
@@ -51,3 +51,15 @@ def validate_game_event(court, ball_game, min_number_of_players, max_number_of_p
     @staticmethod
     def is_ball_game_playable_at_court(court, ball_game):
         return CourtBallGame.is_ball_game_playable(court, ball_game)
+
+    def create_with_admin(self, player, ball_responsible):
+        from game_event_player.models import GameEventPlayer
```

Comment:
Perhaps we should move the import to the top of the page? In my opinion it would reduce duplicated code (same model imported in `delete_game_event`).